lib/validators/specValidator.ts

```diff
@@ -52,7 +52,7 @@ export interface ValidationResultScenarios {
 }
 
 export interface SpecValidationResult extends ModelValidation {
-  validityStatus: unknown
+  validityStatus: boolean
```

Comment:
can we leave it as unknown, ill update today in another PR with another breaking change

Reply:
No, we need to change in this PR so we can use the flag.

Reply:
We need it as `Boolean` for this PR.